composer/helpers/v2/src/UpdateChecker.php

```diff
@@ -59,15 +60,20 @@ public static function getLatestResolvableVersion(array $args): ?string
             $composer->getAutoloadGenerator()
         );
 
+        $composer->getEventDispatcher()->setRunScripts(false);
+
         // For all potential options, see UpdateCommand in composer
         $install
             ->setUpdate(true)
             ->setInstall(false)
             ->setDevMode(true)
             ->setUpdateAllowTransitiveDependencies(Request::UPDATE_LISTED_WITH_TRANSITIVE_DEPS)
             ->setDumpAutoloader(false)
-            ->setRunScripts(false)
-            ->setIgnorePlatformRequirements(false);
+            ->setPlatformRequirementFilter(PlatformRequirementFilterFactory::fromBoolOrList(false));
```

Comment:
😍 for fixing this, as a non-PHP expert, I would have muddled for a while before realizing it needed another import...